platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/admin/ConfigurationAdminMigratable.java

```diff
@@ -174,6 +187,19 @@ private boolean isASource(String factoryPid) {
   // entries must be updated to the latest versions of themselves in order to restore correctly
   private ImportMigrationConfigurationAdminEntry updateNecessaryConfigurationAdminEntries(
       ImportMigrationConfigurationAdminEntry entry) {
+    List<Function<ImportMigrationConfigurationAdminEntry, ImportMigrationConfigurationAdminEntry>>
+        filters = new ArrayList<>();
+    filters.add(this::updateSessionConfiguration);
+    filters.add(this::updateMetacardValidityFilterPluginConfiguration);
```

Comment:
❗️ Move the creation of that list and additions to it as part of the constructor of this class and only keep the rest of the method here.